site/_config.yml

```diff
@@ -25,9 +25,8 @@ pulsar_repo: https://github.com/apache/incubator-pulsar/tree/master
 baseurl: /
 destination: ../generated-site/content
 
-versions:
-- 1.18
-latest: 1.18
+current_version: 1.19.0-incubating
```

Comment:
That's correct.

Well, in addition to "snapshot" the docs in the `asf-site` branch, by copying the `docs/latest` directory into `docs/v1.20.0`.